db/migrations/2022_05_23_000000_increase_tshift_field_width.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+
+/**
+ * To allow for larger key names such as "2XL-G"
+ */
+class IncreaseTshiftFieldWidth extends Migration
+{
+    /**
+     * Run the migration
+     */
+    public function up()
+    {
+        if (!$this->schema->hasTable('GroupPrivileges')) {
```

Comment:
That check should be irrelevant as only a new table is  changed (and if used the later changed table should be referenced).